store/sqlstore/post_store.go

```diff
@@ -34,6 +34,7 @@ const (
 
 var lastPostTimeCache = utils.NewLru(LAST_POST_TIME_CACHE_SIZE)
 var lastPostsCache = utils.NewLru(LAST_POSTS_CACHE_SIZE)
+var maxPostSizeCached *int32
```

Comment:
Could I get you to put this in the store struct itself rather than add another global variable? (Bonus points if you move these others too.)

Reply:
Yes, absolutely. I shouldn't have shied away from this in the first place :)